CHANGELOG.md

```diff
@@ -46,6 +46,8 @@ and this project adheres to
   - [#1457](https://github.com/iovisor/bpftrace/pull/1457)
 
 #### Tools
+- biosnoop: Added output of disk name, offset and length
+  - Also print time and startup to correlate with the offset time
```

Comment:
```suggestion
  - [#1460](https://github.com/iovisor/bpftrace/pull/1460)
```

The link to the PR can only be added after opening it, a bit annoying but it helps to create a useful changelog :) (please squash this into your current commit)

Reply:
Done